<launch>
  <!-- start joystick -->
	<node pkg="joy" type="joy_node" name="joy_on_robot">
		<param name="dev" value="/dev/input/js0" />
	</node>




		<param name="max_vx_run" value="0.5"/>
		<param name="max_vw_run" value="0.5"/>
		<param name="max_vy_run" value="0.0"/>
		<param name="max_vx" value="0.5"/>
		<param name="max_vw" value="0.5"/>
		<param name="max_vy" value="0.0"/>

		<param name="axis_vw" type="int" value="2" />
		<param name="axis_vx" type="int" value="1" /> 

		<param name="deadman_button" type="int" value="5" /> 
		<param name="run_button" type="int" value="5" /> <!--also the trigger button-->

	<!-- run the teleop node to send movement velocities to the pioneer -->
	<node pkg="p2os_teleop" type="p2os_teleop" name="p2os_teleop" >

	</node>
</launch>

